src/index.js

```diff
@@ -22,11 +24,14 @@ app.use(cors());
 
 app.use(require('morgan')('dev'));
 
-app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
 app.use('/docs', swaggerUi.serve, swaggerUi.setup(swaggerdoc));
 
 Routes(app);
+// registered routes
```

Comment:
Please remove all comments

Reply:
Thank you for this review. This has been resolved.